### PR TITLE
Refactor github issue templates.

### PR DESCRIPTION
**Pull request by HumairAK**
This commit does the following:
1. Re-Order templates so they appear in the order users are more
   likely to click them.
2. Switch out 'task' and 'feature' issues in exchange for
   'user story' template.
3. Refactor remaining markdown issue templates to github forms.

### CODE REVIEW

**Comment by HumairAK** [comment]
Old: https://github.com/operate-first/support/issues/new/choose

New: https://github.com/HumairAK/odh-moc-support/issues/new/choose

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ---
 repos:
-  - repo: git://github.com/Lucas-C/pre-commit-hooks
```
Comment (larsks) [inline .pre-commit-config.yaml:3]: I would split these changes into a separate commit (because they're not really related to the other changes here and deserve their own description).

File: .github/ISSUE_TEMPLATE/4_onboarding_to_cluster.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 name: Onboard to a cluster
-description: Get access to an Operate-First managed cluster.
+description: Get access to a namespace in an Operate-First managed cluster.
```
Comment (larsks) [inline .github/ISSUE_TEMPLATE/4_onboarding_to_cluster.yaml:2]: s/Get/Request/ to make this read like other descriptions.

File: .github/ISSUE_TEMPLATE/us_kafka_user_topic_request.yaml [hunk 3]
```diff
@@ -0,0 +1,68 @@
+name: "User support: Kafka Topic/User request"
+description: Template for users to request Kafka Topics and User credentials/certs.
```
Comment (larsks) [inline .github/ISSUE_TEMPLATE/us_kafka_user_topic_request.yaml:2]: Perhaps delete everything before "request" to make this read like other descriptions: "Request Kakfa topics and user credentials"

File: .github/ISSUE_TEMPLATE/2_user_story.yaml [hunk 4]
```diff
@@ -0,0 +1,30 @@
+name: User Story
+description: Add a user story for the Operate First Cloud Instance. Use this to request new features and improvements.
+title: "As a <persona>, I <want to>, <so that>."
```
[the remark this answers is not in the document]
Comment (HumairAK) [reply .github/ISSUE_TEMPLATE/2_user_story.yaml:3]: What sort of example did you have in mind? Like a link to a markdown/page where we list out these personas?

**Review by dystewart — COMMENTED** [review]
/lgtm

File: .github/ISSUE_TEMPLATE/2_user_story.yaml (hunk 4 above)
[the remark this answers is not in the document]
Comment (HumairAK) [reply .github/ISSUE_TEMPLATE/2_user_story.yaml:3]: @larsks

[the remark this answers is not in the document]
Comment (larsks) [reply .github/ISSUE_TEMPLATE/2_user_story.yaml:3]: Maybe something like that. 

I feel as if "User story" and "persona" are jargon words that aren't going to be familiar to folks without prior software developer experience. If this is meant for the general community, I would consider renaming "user story" to "feature request" and drop "As a <persona>" from the example title.

But I don't feel so strongly about this that I'm going to fight you on it :). This PR is generally just fine, and I'm not going to grumble if it merges as-is.
